### DISCUSSION TITLE
Dịch Progressive Growing of GANs

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#166] [Terminology] [answered]
chị @nguyenhoa93 đang để từ này là "Sự tiến triển của GAN". 
Mình nghĩ "phát triển dần dần GAN" nghe hợp hơn.

Xin ý kiến mọi người nhé @mlbvn/handson-ml

**Comment by cuongvng** [comment, marked as answer]
`sự phát triển dần của GAN`?
